src/app/common/interfaces/createAssociation.ts

```diff
@@ -0,0 +1,5 @@
+import { CreateApartment } from './createApartment';
+export interface CreateAssociation {
```

Comment:
asta nu are ce cauta in common... o folosesti si in alte module sau doar in create-association?
deci o pui in interfaces in modulul in care e folosit